### ISSUE TITLE
A query about theoretical total input of spike-ins

### ISSUE THREAD

**Issue by WT215** [LuyiTian/sc_mixology#4]
Hi,

For the code:
```
cms_095046 <- read.delim("~/cms_095046.txt", stringsAsFactors = FALSE)
SpikeInfo = data.frame(ERCCID=cms_095046$ERCC.ID, count=cms_095046$concentration.in.Mix.1..attomoles.ul.)
SpikeInfo = SpikeInfo[SpikeInfo$ERCCID %in% rownames(ERCC_dat),]
rownames(SpikeInfo) = SpikeInfo$ERCCID
SpikeInfo[,2] = SpikeInfo[,2]/1000
```
May I ask can I calculated the theoretical total input of spike-ins as `sum(SpikeInfo$count)`? If so, is it the same for both `CELL_seq2` and `Sort-seq` datasets?

I would like to divide the total number of observed spike-ins by that number to obtain `capture efficiencies` which should range between 0 and 1.

Thank you very much!

Best wishes,
Wenhao

**Comment by LuyiTian** [comment]
Hi Wenhao,

I have added the ERCC annotation: https://github.com/LuyiTian/CellBench_data/tree/master/misc. 

It should be used with care though. We have noticed the calculated capture efficiency is much lower than we expected and we thought it might be caused by ERCC degradation because there are a couple of freeze-thaw cycles and dilution. Also, we used a very high dilution factor, so as you might see many ERCC genes have a theoretical input lower than one, so cannot be detected even with 100% capture efficiency. personally, I found the methods that use ERCC to separate technical variation do not perform very well, such as BASiCS. 

The last thing is that the RNA mixture data were generated from pure RNA instead of cells, where ERCC spike-in behaved well. But the ERCC have more variation in real single-cell data, which might be related to cellular content, as ERCC have short polyA and no secondary structures or attached proteins, making them more vulnerable than cellular mRNA.
